CMakeLists.txt

```diff
@@ -264,33 +273,32 @@ find_opt_library_and_header(EDIT_PATH histedit.h EDIT_LIB edit HAVE_EDIT)
 ########################################################################
 
 macro(add_ledger_library_dependencies _target)
-  target_link_libraries(${_target} ${MPFR_LIB})
-  target_link_libraries(${_target} ${GMP_LIB})
+  target_link_libraries(${_target} PRIVATE ${MPFR_LIB} ${GMP_LIB})
   if (HAVE_EDIT)
-    target_link_libraries(${_target} ${EDIT_LIB})
+    target_link_libraries(${_target} PUBLIC ${EDIT_LIB})
```

Comment:
When using `Python_add_library` (see below) CMake complained about use of keyword vs. plain arguments; adding the `PUBLIC` and `PRIVATE` scope keywords here fixed the error.

ℹ️ `${EDIT_LIB}` needs to be public as the `${LEDGER_CLI_SOURCES}` use it and the linker will fail otherwise.